### PR TITLE
TF-8784/Add workflow cron job to merge staging into main every other week

### PR DESCRIPTION
**Pull request by LeoFuna**
### Ticket(s)

- [8784](https://airtable.com/appfJZShN8K4tcWHU/pagXdnDYi0tVl4u8R?HjEAY=rec1pUwPFHLipqUm1)

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Description

Add a new workflow that handle `staging into main` process from TF team every other week.

This workflow has some jobs before merging to do some basics verification on code to be merged.

If something fail, a new issue is created to inform that something need to be done.

`IMPORTANT NOTE:` We need to create some env before merge this 

#### Process on `Success`
image

#### Process on `Fail`
image

#### Automated `Issue` created
image

### Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [x] I have filled this PR out fully, and cleaned up any extraneous items so that it is clear and easy to understand

### CODE REVIEW

File: .github/workflows/scheduled_merge.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4.1.1
+      with:
+        persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal access token.
+        fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
+
+    - name: Configure user
+      run: |
+        git remote set-url origin "https://${{ secrets.PAT_TOKEN }}@github.com/funas-org/cv-tests.git"
```
Comment (e-papanicolas) [inline .github/workflows/scheduled_merge.yml:89]: You can go ahead and add the `PAT_TOKEN` to Github actions secrets by logging into the cv-dev account, and then I think we also need to update the remote origin url.

File: .github/workflows/scheduled_merge.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+on:
+  workflow_dispatch:
+    inputs:
+      run_job:
+        description: 'Execute manually'
+        required: true
+        default: 'yes'
+  schedule:
+    - cron: '0 17 */15 * *'  # execute every 15 days at 17:00 UTC
```
Comment (e-papanicolas) [inline .github/workflows/scheduled_merge.yml:11]: I think that every 14 days might be better, this way it lands on the same every-other-week schedule as our TF team meetings, just for the sake of remembering exactly when it is happening!

**Review by e-papanicolas — CHANGES_REQUESTED** [review]
@LeoFuna this is so cool! I'm so excited to have this up and running.

File: .github/workflows/scheduled_merge.yml (hunk 1 above)
Comment (LeoFuna) [reply .github/workflows/scheduled_merge.yml:89]: Ohh great catch! I forgot to change that

File: .github/workflows/scheduled_merge.yml (hunk 2 above)
Comment (LeoFuna) [reply .github/workflows/scheduled_merge.yml:11]: Perfect! I will change this :heavy_check_mark:

**Comment by LeoFuna** [comment]
I created necessaries `envs`.
So everything is set up :heavy_check_mark:

File: .github/workflows/scheduled_merge.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ on:
         required: true
         default: 'yes'
   schedule:
-    - cron: '0 17 */15 * *'  # execute every 15 days at 17:00 UTC
+    - cron: '0 17 */14 * *'  # execute every 15 days at 17:00 UTC
```
Comment (e-papanicolas) [inline .github/workflows/scheduled_merge.yml:11]: @LeoFuna Will you please adjust the comment that still says 15 days 🙏🏼

**Review by e-papanicolas — APPROVED** [review]

**Pull request merged**